### PR TITLE
Login feature

### PR DESCRIPTION
**Pull request by SibuMbi**
Please review.

### CODE REVIEW

File: .vscode/settings.json
Comment (agorbacheva-y) [inline .vscode/settings.json:1]: i'm not sure what this file is for???

File: public/Child.png
Comment (agorbacheva-y) [inline public/Child.png:1]: png not needed. svg of this image is in images folder

File: src/loginFeature.html
Comment (agorbacheva-y) [inline src/loginFeature.html:1]: no need for html file

File: public/Child.png
Comment (SibuMbi) [reply public/Child.png:1]: Removed

File: .vscode/settings.json
Comment (SibuMbi) [reply .vscode/settings.json:1]: Folder removed.

File: src/loginFeature.html
Comment (SibuMbi) [reply src/loginFeature.html:1]: File removed.

File: src/loginfeature.css
Comment (agorbacheva-y) [inline src/loginfeature.css:1]: we should write our own css to follow the figma design

File: src/loginFeature.js
Comment (agorbacheva-y) [inline src/loginFeature.js:1]: i think this file needs to start with a capital letter?

**Review by agorbacheva-y — COMMENTED** [review]
can we continue to work on this together before merging?

**Pull request merged**